.github/workflows/Java_CI.yaml

```diff
@@ -1,16 +1,31 @@
 name: Java CI
 
 on:
+  pull_request:
+    branches:
+      - wrims-devops
   push:
-    branches: [ Feature/wrims-core ]
+    branches:
+      - Feature/wrims-core
```

Comment:
You don't need to hard-code branches here. The build for `Feature/wrims-core` will run through CI/CD through the pull_request condition above